src/Analyzers/CSharp/Tests/AddAccessibilityModifiers/AddAccessibilityModifiersFixAllTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.AddAccessibilityModifiers;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AddAccessibilityModifiers
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
+    public class AddAccessibilityModifiersFixAllTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
Let us avoid using the legacy test framework for these new tests. See https://github.com/dotnet/roslyn/blob/main/src/Analyzers/CSharp/Tests/AddBraces/AddBracesFixAllTests.cs for example FixAll tests based on the new Microsoft.CodeAnalysis.Testing library.

Reply:
Add braces seems to use the same implementation as this?  It's a partial
https://github.com/dotnet/roslyn/blob/main/src/Analyzers/CSharp/Tests/AddBraces/AddBracesTests.cs#L20

Reply:
Tried a few variations of the following test using the new framework, but I don't think its possible.  The new test framework never calls into `IFixAllSpanMappingService`.  Seems like that exists on a higher layer than these tests.  This test always passes even if I remove my fix.

```
[Fact, WorkItem("https://github.com/dotnet/vscode-csharp/issues/6611")]
[Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
public async Task TestFixAll()
{
    var source = """
        using System;
        Console.WriteLine("Hello, World!");

        class [|Program|]
        {
        }
        """;

    var fixedSource = """
        using System;
        Console.WriteLine("Hello, World!");

        internal class Program
        {
        }
        """;

    var test = new VerifyCS.Test
    {
        TestCode = source,
        FixedCode = fixedSource,
        //Options =
        //    {
        //        { CSharpCodeStyleOptions.UnusedValueAssignment, (UnusedValuePreference)option },
        //    },
        LanguageVersion = LanguageVersion.CSharp10,
        // This is not compilable as-is - we have a duplicate Program definition.
        CompilerDiagnostics = CompilerDiagnostics.None,
        BatchFixedCode = fixedSource,
    };

    await test.RunAsync();
}
```

Reply:
Ah, my bad. I presume the new testing library doesn’t yet support FixAllInContainingType and FixAllInContainingMember scopes.